pkg/controller/backup_controller.go

```diff
@@ -436,6 +436,15 @@ func (c *backupController) prepareBackupRequest(backup *velerov1api.Backup) *pkg
 	request.Annotations[velerov1api.SourceClusterK8sMajorVersionAnnotation] = c.discoveryHelper.ServerVersion().Major
 	request.Annotations[velerov1api.SourceClusterK8sMinorVersionAnnotation] = c.discoveryHelper.ServerVersion().Minor
 
+	// Add namespaces with label velero.io/exclude-from-backup=true into request.Spec.ExcludedNamespaces
+	// Essentially, adding the label velero.io/exclude-from-backup=true to a namespace would be equivalent to setting spec.ExcludedNamespaces
+	namespaces, excludeLabel := corev1api.NamespaceList{}, "velero.io/exclude-from-backup"
+	if err := c.kbClient.List(context.Background(), &namespaces, kbclient.MatchingLabels{excludeLabel: "true"}); err == nil {
```

Comment:
This is not a validation error strictly speaking but I don't think it's right to eat the error.
Would there be a problem if we set it into `request.Status.ValidationErrors` and make it fail early and explicitly?

Reply:
Seems it's better to  append the potential err into request.Status.ValidationErrors with explicit err message.